scalding-core/src/main/scala/com/twitter/scalding/Mode.scala

```diff
@@ -118,7 +127,9 @@ trait HadoopMode extends Mode {
         asMap - jarKey
       case None => asMap
     }
-    new HadoopFlowConnector(finalMap.asJava)
+    val clazz = Class.forName(jobConf.get("cascading.flow.connector.class", "cascading.flow.hadoop.HadoopFlowConnector"))
```

Comment:
Should probably try catch around this and print a good error message, why we are loading it and that it comes from that field in the jobconf